qq

```diff
@@ -99,7 +99,7 @@ __qq () {
   else
     # local INPUT=$@
     # QUERY="'kind:text AND filename:.$NOTESEXT AND filename:$NOTESPRE$(__qq_query_include_all "${*%\?}")${EXCLUDEQUERY}'"
-    QUERY="mdfind -onlyin '$NOTESDIR' -interpret '(kind:text OR kind:markdown) AND filename:.$NOTESEXT AND filename:$NOTESPRE$(__qq_query_include_all "${*%\?}")${EXCLUDEQUERY}'"
```

Comment:
the inclusion of `.` before `$NOTESEXT` prevented `mdfind` from showing results when invoked from command line